mortgageAPI.yaml

```diff
@@ -1933,6 +1933,11 @@ components:
               enum: [none, minergie, minergie-p, minergie-eco, minergie-p-eco]
               description: 'The minergie standard'
               example: 'minergie-p'
+            luxusFlag:
+              type: string
+              enum: [0, 0.5, 1.0]
```

Comment:
Not completely certain, but I _think_ this should be strings, i.e., `['0', '0.5', '1.0']`

Reply:
to my knowledge both ways are ok.